frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.tsx

```diff
@@ -82,6 +83,9 @@ const DataResiliency: React.FC<DashboardItemProps> = ({
     <DashboardCard className="ceph-data-resiliency__dashboard-card">
       <DashboardCardHeader>
         <DashboardCardTitle>Data Resiliency</DashboardCardTitle>
+        <DashboardCardHelp>
+          Data Resiliency presents status of data replication and rebalancing operations.
```

Comment:
can we follow the same pattern? define a `const infoText` and then use it inside `DashboardCardHeader` component. The same you did for object-data-reduction-card.

Reply:
@cloudbehl here the text is small, easy to read and looks good inside tags :D

Reply:
> @cloudbehl here the text is small, easy to read in a flow and looks good inside tags :D

Reply:
but my idea is we should change the constant rather than changing something inside the component. Increases chances for getting bugs.

Reply:
alright! 
what I saw this as a hard-coded value that is not going to change just like the names of all cards. I have written `const info text` in data reduction to enhance readability but I have no issues in changing it as per your suggestion :)